CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Upcoming
 
+# v0.5.3
```

Comment:
```suggestion
# v0.6.0
```

I would say this is more of a minor release since we added Zarr support. (If we decide 0.5.3 instead, then we'll need to update the version in `pyproject.toml`)

Reply:
that makes sense! Thanks, I missed that it had been updated in the `pyproject.toml` already.